src/userflow.ts

```diff
@@ -308,7 +308,7 @@ if (!userflow) {
   ) {
     userflow![method] = function () {
       var args = Array.prototype.slice.call(arguments)
-      userflow!.load()
+      userflow!.load().then(() => {}, () => {})
```

Comment:
Actually with the current approach users cannot really control it nor adapt their handling to it. Maybe it would be great to let the caller of the method deal with the error (catch or not it). I can adapt my code to come up with something with that idea in mind

Reply:
It could be as simple as:

```js
  var stubVoid = function (
    // eslint-disable-next-line es5/no-rest-parameters
    method: ConditionalKeys<Userflow, (...args: any[]) => void>
  ) {
    userflow![method] = function () {
      var args = Array.prototype.slice.call(arguments)
      var loadPromise = userflow!.load()
      q.push([method, null, args])
      return loadPromise
    } as any
  }
```

Last problem would mostly be around handling the calls to stubVoid that are issued at mount time.

Reply:
Btw, you can still control the loading behaviour.  All the `stub` methods load the userflow if it's not already loaded yet. So, you can manually load the library before initialising it. 

Eg:

```js
import userflow from "userflow.js";

userflow
  .load()
  .then(() => userflow.init('<your-token-here>'))
  .catch((err) => console.log("failed in loading userflow"));
```

Here is a simulation of the same
![load-userflow](https://github.com/user-attachments/assets/953fc4e5-11bb-444d-827c-7ac447477211)

Reply:
@dubzzz let us know if using the load helped in catching the errors for loading. Probably we can add to our documentation to help for these cases.

Reply:
I'll give it a try. Thanks for the answer it seems to do the trick I need